pkg/generate/route.go

```diff
@@ -17,23 +17,25 @@ func DefaultRoute(cr *regopapi.ImageRegistry, p *parameters.Globals) (Template,
 			Kind:       "Route",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      p.DefaultRoute.Name,
+			Name:      cr.ObjectMeta.Name + "-default-route",
 			Namespace: p.Deployment.Namespace,
 		},
 		Spec: routeapi.RouteSpec{
 			To: routeapi.RouteTargetReference{
 				Kind: "Service",
-				Name: p.Deployment.Name,
+				Name: cr.ObjectMeta.Name,
 			},
 		},
 	}
+
+	r.Spec.TLS = &routeapi.TLSConfig{}
+
+	// TLS certificates are served by the front end of the router, so they must be configured into the route,
+	// otherwise the router's default certificate will be used for TLS termination.
 	if cr.Spec.TLS {
-		// TLS certificates are served by the front end of the router, so they must be configured into the route,
-		// otherwise the router's default certificate will be used for TLS termination.
-		r.Spec.TLS = &routeapi.TLSConfig{
-			Termination: routeapi.TLSTerminationReencrypt,
-		}
+		r.Spec.TLS.Termination = routeapi.TLSTerminationReencrypt
 	}
```

Comment:
should be setting the termination to edge otherwise?

Reply:
@bparees As we discussed today for the default route, we always use reencrypt. no ?

Reply:
no, since it's still possible for someone to turn off TLS on the registry service, so we still have to be able to use edge termination on the default route.